packages/lib/reducer.ts

```diff
@@ -1073,6 +1075,10 @@ const reducer = produce((draft: Draft<State> = defaultState, action: any) => {
 			}
 			break;
 
+		case 'NOTE_ID_CONVERTED':
```

Comment:
Without context it'd be very hard to know what this is. Maybe let's rename to "NOTE_HTML_TO_MARKDOWN_DONE"

**And also the state variable.**